### ISSUE TITLE
Millisecond fraction of seconds always interpreted as microseconds 

### ISSUE THREAD

**Issue by agijsberts** [zozzz/yapic.json#11]
When the length of the fraction of seconds is not exactly 6, the current implementation implicitly fills the remaining spaces with leadings zeros (`.123` -> `.000123`), whereas trailing zeros would probably make more sense (`.123` -> `.123000`). As a specific example, milliseconds in the following example become microseconds:
```python
>>> from yapic import json
>>> json.loads('{"timestamp":"2021-04-07T20:23:44.123Z"}')
{'timestamp': datetime.datetime(2021, 4, 7, 20, 23, 44, 123, tzinfo=datetime.timezone.utc)}
>>> _['timestamp'].isoformat()
'2021-04-07T20:23:44.000123+00:00'
```

**Comment by bmoscon** [comment]
@zozzz - any idea of when a release might be available with this change?

**Comment by zozzz** [comment]
Already available 1.6.3
